tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/APIView/APIViewRevisionTool.cs

```diff
@@ -0,0 +1,216 @@
+using System.CommandLine;
+using System.ComponentModel;
+using Azure.Sdk.Tools.Cli.Commands;
+using Azure.Sdk.Tools.Cli.Models;
+using Azure.Sdk.Tools.Cli.Models.Responses;
+using Azure.Sdk.Tools.Cli.Services;
+using ModelContextProtocol.Server;
+
+namespace Azure.Sdk.Tools.Cli.Tools.APIView;
+
+[McpServerToolType]
+[Description("APIView revision operations including comments and content")]
+public class APIViewRevisionTool : MCPMultiCommandTool
+{
+    // Sub-command constants
+    private const string GetComments = "get-comments";
+    private const string GetContent = "get-content";
+
+    public override CommandGroup[] CommandHierarchy { get; set; } = [SharedCommandGroups.APIView];
+
+    private static readonly string[] validContentTypes = ["Text", "CodeFile"];
+
+    private readonly IAPIViewService _apiViewService;
+    private readonly ILogger<APIViewRevisionTool> _logger;
+
+    private readonly Option<string> environmentOption = new("--environment")
+    {
+        Description = "The APIView environment (defaults to production)",
```

Comment:
Instead of taking this as a param, can you try to use an env variable to override the environment? If it is set then connect to that env otherwise connect to prod.  We can avoid this extra param. Only ppl who needs to access other than prod is APIView MCP dev and end user will always needs to connect to prod.